packages/api/amplify_api_ios/ios/Classes/Api.h

```diff
@@ -0,0 +1,4 @@
+#import <Flutter/Flutter.h>
+
+@interface Api : NSObject<FlutterPlugin>
```

Comment:
On the record again, I really dislike these renames

Reply:
@haverchuck what are your thoughts on renaming to just Api?  We discussed this briefly in the past how the Auth Cognito naming scheme would require renaming the other modules as well.  Since Api is not AWS specific, that would lead us to the Api name scheme we have now. 

Should it be AmplifyApi instead perhaps?  What are your other suggestions @dnys1 that would be most consistent with our other module renames (Auth_Cognito, Analytics_Pinpoint, etc)

Reply:
Yes, I would be happy with anything more descriptive than Api: `AmplifyApi`, `AmplifyApiPlugin`

Reply:
AmplifyApi